### PR TITLE
Simplify APOC auto-cherry-picking (4.2)

### PR DESCRIPTION
**Pull request by Lojjs**
Stop to auto cherry-pick 4.2 PRs to 4.1

4.1 is not supported anymore

### CODE REVIEW

**Comment by Lojjs** [comment]
[Simplify APOC auto-cherry-picking](https://trello.com/c/kfyRimWe/57-simplify-apoc-auto-cherry-picking)

**Review by conker84 — APPROVED** [review]
LGTM, waiting for @ncordon approval

**Review by ncordon — APPROVED** [review]

**Pull request merged**
